eox_nelp/views.py

```diff
@@ -34,3 +40,26 @@ def info_view(request):
         json.dumps(response_data),
         content_type="application/json"
     )
+
+
+class NelpMFEConfigView(MFEConfigView):
+    """
+    Provides an API endpoint to get the MFE_CONFIG from site configuration with
+    nelp extra fields.
+    """
+
+    @method_decorator(cache_page(settings.MFE_CONFIG_API_CACHE_TIMEOUT))
+    def get(self, request):
+        """
+        Return the MFE configuration by NELP(adding custom nelp fields).
+        """
+        base_get_response = super().get(request)
```

Comment:
all this logic is based on a status 200, so please verify that  the super method returns that status otherwise  just return the same response

Reply:
[7444ad5](https://github.com/eduNEXT/eox-nelp/pull/17/commits/7444ad5282b135b8f1397c78415911a52e99a24e)